lib/trento_web/controllers/fallback_controller.ex

```diff
@@ -3,59 +3,38 @@ defmodule TrentoWeb.FallbackController do
 
   alias TrentoWeb.ErrorView
 
-  def call(conn, {:error, {:bad_request, errors = %{}}}) do
-    conn
-    |> put_status(:bad_request)
-    |> render_errors(errors)
-  end
-
-  def call(conn, {:error, {:bad_request, reason}}) do
-    conn
-    |> put_status(:bad_request)
-    |> render_error(reason)
-  end
-
-  def call(conn, {:error, {:unprocessable_entity, reason}}) do
+  def call(conn, {:error, {:unprocessable_entity, error}}) do
     conn
     |> put_status(:unprocessable_entity)
-    |> render_error(reason)
+    |> put_view(ErrorView)
+    |> render(:"422", error: error)
   end
 
-  def call(conn, {:error, {:internal_error, reason}}) do
+  def call(conn, {:error, {:internal_error, detail}}) do
     conn
     |> put_status(:internal_server_error)
-    |> render_error(reason)
+    |> put_view(ErrorView)
+    |> render(:"500", detail: detail)
   end
 
-  def call(conn, {:error, {:not_found, reason}}) do
+  def call(conn, {:error, {:not_found, detail}}) do
     conn
     |> put_status(:not_found)
-    |> render_error(reason)
+    |> put_view(ErrorView)
+    |> render(:"404", detail: detail)
   end
 
   def call(conn, {:error, :invalid_credentials}) do
     conn
     |> put_status(:unauthorized)
-    |> put_view(TrentoWeb.SessionView)
-    |> render("invalid_credentials.json")
-  end
-
-  def call(conn, {:error, :unauthorized}) do
-    conn
-    |> put_status(:unauthorized)
-    |> put_view(TrentoWeb.SessionView)
-    |> render("unauthorized.json")
-  end
-
-  defp render_error(conn, reason) do
-    conn
     |> put_view(ErrorView)
-    |> render("error.json", reason: reason)
+    |> render(:"401")
```

Comment:
Shouldn't we set some `detail` here? Like `Invalid credentials` or something.
Otherwise, how is this rendered?
I see in the tests that the used details is ` Invalid crendetials`, but is this coming from here or the next `call` function?
Maybe this `call` is not even used

Reply:
This actually should be removed, because it never happens. Note that this is a temporary fallback controller refractor and we are not using the fallback controller properly yet.